python/cugraph/cugraph/tests/mg/test_mg_graph.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Can you add an additional test to this file (and sg as well) to ensure renumbering works as expected when edge ids, edge types, and weights are passed in as the edge attributes?